scripts/hid_configurator/configurator_cli.py

```diff
@@ -93,6 +103,9 @@ def parse_arguments():
         sp_commands.add_parser('fwinfo', help='Obtain information about FW image')
         sp_commands.add_parser('fwreboot', help='Request FW reboot')
 
+        if device_name == 'gaming_mouse':
```

Comment:
Maybe we could add this option anyway and just later on check if device supports it ?